src/app/screens/sendBrc20/index.tsx

```diff
@@ -152,29 +152,30 @@ function SendBrc20Screen() {
         selectedAccount?.id ?? 0,
         seedPhrase,
         network.type,
-      ).catch((err) => {
-        if (Number(err) === ErrorCodes.InSufficientBalance) {
-          setAmountError(t('SEND.ERRORS.INSUFFICIENT_BALANCE'));
-        } else if (Number(err) === ErrorCodes.InSufficientBalanceWithTxFee) {
-          setAmountError(t('SEND.ERRORS.INSUFFICIENT_BALANCE_FEES'));
-        } else setAmountError(err.toString());
-      });
+      );
       navigate('/confirm-inscription-request', {
         state: {
-          brcContent: order.inscriptionRequest.files[0].dataURL,
+          // !NOTE: Typing below is broken. Not going to fix as this page is deprecated.
```

Comment:
Let's take this opportunity to remove the unused code.

Reply:
Will do 😄